recipes/spreadgltools/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name ="SpreadGLtools" %}
+{% set version ="1.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/GuyBaele/SpreadGL.git
+  git_rev: main
```

Comment:
This is not good!
Please use a Git tag instead or at least a commit hash. `main` is a moving target

Reply:
Agreed, chatting devs to make a release as there is none atm.